pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -39,8 +39,9 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
 	priorityutil "k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	heap "k8s.io/kubernetes/pkg/scheduler/internal/heap"
```

Comment:
these import aliases are redundant. If your IDE added them, then I don't want to spend effort on it. If you're manually adding them, I think we should remove it.

Reply:
My IDE added this redundant import. Do let me know if I should refactor it?

Reply:
If your IDE added them, I don't want you (or anyone) to waste time manually fixing it. I think `goimports` somewhat recently changed its behavior around adding aliases. Ultimately, it's not doing any harm.

Reply:
If your IDE added it, I don't want you (or anyone) wasting time manually fixing it. I think that `goimports` changed how it does import aliases a while back and it's causing things like this. In the end, it is not causing harm.

Reply:
@misterikkit Ravi told me that he was awaiting some changes to be incorporated from @liggitt 's review. Do you have any suggestions of changes to be made into this PR ?

Reply:
For import aliases, no changes are needed. For my other comment about the feature gate condition, that needs to change.

Reply:
I have removed that condition now.